bin/dipy_csa

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
```

Comment:
@Garyfallidis I might have missed the discussion here. Why removing "/usr/bin/env"? 

`#! /usr/bin/env python` works for me while `#!python` doesn't.  I execute the scripts from `dipy/bin`

Reply:
ok, there is a thread up there about this...

> I mean that if the shebang is only #!python, calling this script doesn't work correctly when working from inside a virtualenv. If the shebang is changed to /usr/bin/env python, it works correctly.

Reply:
Yeap. Make sure you switch into the ```pip install --user -e . ```.